src/main/java/com/hierynomus/msfscc/fileinformation/FileInformationFactory.java

```diff
@@ -480,6 +492,31 @@ private static FileEaInformation parseFileEaInformation(Buffer<?> buffer) throws
         return new FileEaInformation(eaSize);
     }
 
+    /**
+     * 2.4.40 FileStreamInformation
+     */
+    private static FileStreamInformation parseFileStreamInformation(Buffer<?> buffer) throws Buffer.BufferException {
+
+        ArrayList<FileStreamInformationItem> streamList=new ArrayList<>();
```

Comment:
Could you change this to `List<FileStreamInformationItem> streamList=new ArrayList<>();`
Also please apply consistent code formatting. Some spaces missing here.